pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -396,8 +397,10 @@ type PodSpec struct {
 }
 
 type HAProxySpec struct {
-	PodSpec                `json:",inline"`
-	ReplicasServiceEnabled *bool `json:"replicasServiceEnabled,omitempty"`
+	PodSpec                          `json:",inline"`
+	ReplicasServiceEnabled           *bool    `json:"replicasServiceEnabled,omitempty"`
+	ReplicasLoadBalancerSourceRanges []string `json:"replicasLoadBalancerSourceRanges,omitempty"`
```

Comment:
we need to add all new options into cr.yaml

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1196/commits/ee068209f4c2ce61f2274c445aad7b1def0f24f7